ui/admin/app/styles/app.scss

```diff
@@ -452,3 +454,103 @@
     }
   }
 }
+
+// secret editor
+.json-secret-editor {
+  .CodeMirror {
+    height: 210px;
+  }
+
+  .CodeMirror-gutters {
+    background: #31332b;
+  }
+}
+
+.secret-editor {
+  background: #272822;
+  border-radius: 4px;
```

Comment:
Don't we normally use `sizing.rems` instead of direct pixels?

Reply:
Fixed.